governance/polls/Adjust the WBTC Debt Ceiling and Risk Premium - June 29, 2020

```diff
@@ -0,0 +1,35 @@
+---
+title: Adjust the WBTC Debt Ceiling and Risk Premium - June 29, 2020
+summary: Signal your support for increasing the WBTC Debt Ceiling from 10m to 20m. And increasing the WBTC Risk Premium from 1% to 2%.
+discussion_link: https://forum.makerdao.com/t/2824
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Adjust the WBTC Debt Ceiling - June 29, 2020
+
+The Maker Governance Community has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) which the community can use to increase the WBTC Debt Ceiling from 10m to 20m. And to increase the WBTC Risk Premium from 1% to 2%.
+
+This Governance Poll ([FAQ](https://community-development.makerdao.com/governance/governance#is-there-more-than-one-type-of-vote)) will be active for three days beginning on Monday, June 29 at 5 PM UTC, the results of which may inform an Executive Vote ([FAQ](https://community-development.makerdao.com/governance/governance#what-is-continuous-approval-voting)) which will go live on Friday, July 3, at 5 PM UTC.
+
+## Review
+
+Additional context was presented in a [Signal Request](https://forum.makerdao.com/t/2824) on June 13th, 2020. Please review to inform your position before voting.
+
+## Next Steps
+
+* On the Friday following the conclusion of the poll, there will be an Executive Vote asking MKR token holders if they support or reject the change proposed by this Governance Poll.
```

Comment:
I might remove the bullet point (just the *, not the whole sentence). Super minor issue though.

Reply:
Thanks.